crates/router/src/connector/authorizedotnet/transformers.rs

```diff
@@ -994,7 +998,14 @@ impl From<SyncStatus> for enums::RefundStatus {
         match transaction_status {
             SyncStatus::RefundSettledSuccessfully => Self::Success,
             SyncStatus::RefundPendingSettlement => Self::Pending,
-            _ => Self::Failure,
+            SyncStatus::AuthorizedPendingCapture
```

Comment:
Don't they have api specific statuses?

Reply:
They have a common status mapping.

Reply:
We have now separated the status mapping for PSYN and RSYNC.